src/plugins/facebook/facebook-en.hbs

```diff
@@ -7,10 +7,16 @@
 	"tag": "facebook",
 	"parentdir": "facebook",
 	"altLangPrefix": "facebook",
-	"dateModified": "2015-08-03"
+	"dateModified": "2022-10-07"
 }
 ---
 <span class="wb-prettify all-pre hide"></span>
+<section class="alert alert-success">
+	<h2>
```

Comment:
An heading can't be use as a paragraph and a section need to be composed from more two or more phrasing element. - fyi https://html.spec.whatwg.org/multipage/dom.html#kinds-of-content 

Here, we can deduct the heading element is used for style and not to provide semantic. related to SC 1.3.1 and the technique F43

Reply:
Francis asked me to use a heading tag for the alert. Would you prefer if I changed it to a paragraph tag instead?

Reply:
Tu peux avoir un titre court suivi d'un paragraph descriptif.

Reply:
Un titre de quelques mots indiquant la nature de cet alerte.

Reply:
J'ai ajouter un titre avec un paragraph descriptif.